revolution_pi/config.go

```diff
@@ -1,16 +1,13 @@
 package revolution_pi
 
 import (
+	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/utils"
 )
 
+var Model = resource.NewModel("viam-labs", "kunbus", "revolutionpi")
+
 type Config struct {
+	resource.TriviallyValidateConfig
```

Comment:
To be honest, we may want to revisit this module's config in light of how I did the modbus module. In the modbus module I take a map of pin names to addresses. We might want to do that here too.

Reply:
We can do that in a follow up - I might have eng time for you on my plan for this module next cycle.